ocean_lib/data_provider/data_service_provider.py

```diff
@@ -519,11 +406,9 @@ def compute_job_result(
         )
         response = DataServiceProvider._http_method("get", compute_job_result_file_url)
 
-        if not response:
-            raise DataProviderException("No response on job result endpoint.")
-
-        if response.status_code != 200:
-            raise DataProviderException(response.content)
+        DataServiceProviderBase.check_response(
+            response, "jobResultEndpoint", compute_job_result_endpoint
+        )
 
         return response.content
```

Comment:
`response.content` returns bytes here, but the signature of this function requires `Dict[str, Any]`.